packages/ra-supabase-core/src/useSupabaseAccessToken.test.tsx

```diff
@@ -70,7 +90,7 @@ describe.skip('useSupabaseAccessToken', () => {
         });
     });
 
-    test('should redirect users to the provided path if the access token is not present in the URL', async () => {
+    test.skip('should redirect users to the provided path if the access token is not present in the URL', async () => {
```

Comment:
Why?

Reply:
All your test for useSupabaseAccessToken were skip. So i just unskip the ones i managed to get working